### PR TITLE
port sdr_transceiver_emb to STEMlab 122.88-16 SDR

### PR DESCRIPTION
**Pull request by howard0su**
I cherrypick the change from 122-16 branch and add some additional fixes.

No intention to checkin this but for review purpose. after review is finish, I will create a separate folder sdr_transceiver_emb_122_88 to host the code.

### CODE REVIEW

File: projects/sdr_transceiver_emb/block_design.tcl [hunk 1]
```diff
@@ -2,16 +2,16 @@
...
   CLKOUT2_USED true
-  CLKOUT2_REQUESTED_OUT_FREQ 250.0
-  CLKOUT2_REQUESTED_PHASE 157.5
+  CLKOUT2_REQUESTED_OUT_FREQ 245.76
+  CLKOUT2_REQUESTED_PHASE -90.0
   CLKOUT3_USED true
-  CLKOUT3_REQUESTED_OUT_FREQ 250.0
-  CLKOUT3_REQUESTED_PHASE 202.5
+  CLKOUT3_REQUESTED_OUT_FREQ 245.76
+  CLKOUT3_REQUESTED_PHASE -45.0
```
Comment (howard0su) [inline projects/sdr_transceiver_emb/block_design.tcl:14]: not sure if this is correct phase value. Not quite sure the purpose of these.

**Comment by pavel-demin** [comment]
The PLL configuration is the same in all 122_88 projects. Just copy it.
https://github.com/pavel-demin/red-pitaya-notes/blob/master/projects/sdr_transceiver_hpsdr_122_88/block_design.tcl#L1-L19

**Comment by pavel-demin** [comment]
sdr_transceiver_emb_122_88 is now in the master branch.

**Comment by howard0su** [comment]
thank you so much

**Pull request closed without merging**